src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -88,8 +88,8 @@ private static async Task<Build> GetLatestBuildAsync(CancellationToken cancellat
             var builds = await GetBuildsFromTFSAsync(buildClient, definitions, cancellationToken, resultFilter: null);
 
             return (await GetInsertableBuildsAsync(buildClient, cancellationToken, from build in builds
-                    orderby build.FinishTime descending
-                    select build)).FirstOrDefault();
+                                                                                   orderby build.FinishTime descending
```

Comment:
ehhh I understand if you don't bother here but it feels like the arguments to GetInsertableBuildsAsync should be wrapped here.

Reply:
gonna have to change the way this is wrapped because I don't want to revert the outdent only to hit it again with future formats.